feat: set tiles view as default view option

AlexAndBear opened the pull request:
<!--
Thank you for your contribution to OpenCloud!

For reporting potential security issues, contact us via https://opencloud.eu/

Please follow these guidelines while opening a pull request:

- Set appropriate labels
- Assign it to yourself.
- Choose at least one reviewer.
-->

## Description

<!--- Describe the subject of the pull request in detail, if appropriate add screenshots  -->

## Related Issue

<!--- If you are fixing a bug, please ensure there's an issue detailing the steps to reproduce it. -->
<!--- Link the issue here: -->

- Fixes #1160 
## How Has This Been Tested?

<!--- Provide a brief description of how you tested your changes. -->
<!--- Include your testing environment and the tests you ran. -->

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

<!--- What types of changes does your code introduce? Mark an x in all the applicable boxes: -->

- [ ] Bugfix
- [ ] Enhancement (a change that doesn't break existing code or deployments)
- [ ] Breaking change (a modification that affects current functionality)
- [ ] Technical debt (addressing code that needs refactoring or improvements)
- [ ] Tests (adding or improving tests)
- [ ] Documentation (updates or additions to documentation)
- [ ] Maintenance (like dependency updates or tooling adjustments)

AlexAndBear commented on tests/e2e/support/objects/app-files/resource/actions.ts:
```diff
@@ -835,11 +836,16 @@ export const selectOrDeselectResources = async (args: selectResourcesArgs): Prom
+      }
     } else if (await resourceCheckbox.isChecked()) {
-      await resourceCheckbox.uncheck()
+      await resourceCheckbox.click()
```
same

AlexAndBear commented on packages/web-pkg/src/components/FilesList/ResourceTiles.vue:
```diff
@@ -59,7 +59,7 @@
           :resource-icon-size="resourceIconSize"
           :draggable="dragDrop"
-          :lazy="lazy"
+          :lazy="false"
```
we need to make this configurable for e2e tests

ScharfViktor commented:
@AlexAndBear I fixed this step
```
And "Alice" copies the following resource using keyboard                                   # file:/Users/v.scharf/Work/opencloud/web/tests/e2e/cucumber/steps/ui/resources.ts:95
        | resource            | to          |
        | KeyboardExample.txt | PARENTCopy3 |
```

but next step is failed
 ```
✘ And "Alice" moves the following resource using drag-drop                                   # file:/Users/v.scharf/Work/opencloud/web/tests/e2e/cucumber/steps/ui/resources.ts:95
        | resource     | to          |
        | dragDrop.txt | PARENTCopy2 |
```

it seems that we cannot move file to folder using drag-drop. (moving folder to folder works)

ScharfViktor commented:
https://github.com/user-attachments/assets/851aeb83-6728-4a40-a018-45915ac7f394

AlexAndBear commented:
> @AlexAndBear I fixed this step
> 
> ```
> And "Alice" copies the following resource using keyboard                                   # file:/Users/v.scharf/Work/opencloud/web/tests/e2e/cucumber/steps/ui/resources.ts:95
>         | resource            | to          |
>         | KeyboardExample.txt | PARENTCopy3 |
> ```
> 
> but next step is failed
> 
> ```
> ✘ And "Alice" moves the following resource using drag-drop                                   # file:/Users/v.scharf/Work/opencloud/web/tests/e2e/cucumber/steps/ui/resources.ts:95
>        | resource     | to          |
>        | dragDrop.txt | PARENTCopy2 |
> ```
> 
> it seems that we cannot move file to folder using drag-drop. (moving folder to folder works)

It looks like that this issue only applies to files that have a preview, moving e.G PDF files (which doesn't have a preview) do work

AlexAndBear commented:
> @AlexAndBear I fixed this step
> 
> ```
> And "Alice" copies the following resource using keyboard                                   # file:/Users/v.scharf/Work/opencloud/web/tests/e2e/cucumber/steps/ui/resources.ts:95
>         | resource            | to          |
>         | KeyboardExample.txt | PARENTCopy3 |
> ```
> 
> but next step is failed
> 
> ```
> ✘ And "Alice" moves the following resource using drag-drop                                   # file:/Users/v.scharf/Work/opencloud/web/tests/e2e/cucumber/steps/ui/resources.ts:95
>        | resource     | to          |
>        | dragDrop.txt | PARENTCopy2 |
> ```
> 
> it seems that we cannot move file to folder using drag-drop. (moving folder to folder works)

fixed

AlexAndBear commented on packages/web-pkg/src/components/FilesList/ResourceTiles.vue:
```diff
@@ -240,6 +242,9 @@ const viewSizeCurrent = computed(() => {
 })
 
+// Disable lazy loading during E2E tests to avoid having to scroll in tests
+const areTilesLazy = (window as any).__E2E__ === true ? false : lazy
```
😱
